python/ray/autoscaler/_private/monitor.py

```diff
@@ -336,10 +336,13 @@ def _run(self):
             try:
                 if self.stop_event and self.stop_event.is_set():
                     break
+                gcs_request_start_time = time.time()
                 self.update_load_metrics()
+                gcs_request_time = time.time() - gcs_request_start_time
```

Comment:
This technically includes other logic? do you think that's negligible? (alternatively, we can return some stats from self. update_load_metrics API)

Reply:
I think it's negligible (the other logic just updates some dicts).